test/fit.jl

```diff
@@ -367,10 +367,18 @@ end
 
 @testset "Testing fit for Laplace" begin
     for func in funcs, dist in (Laplace, Laplace{Float64})
-        d = fit(dist, func[2](dist(5.0, 3.0), N + 1))
+        x = func[2](dist(5.0, 3.0), N + 1)
+        w = func[1](N + 1)
+
+        d = fit(dist, x)
         @test isa(d, dist)
         @test isapprox(location(d), 5.0, atol=0.02)
         @test isapprox(scale(d)   , 3.0, atol=0.03)
+
+        dw = fit(dist, x, w)
+        @test isa(dw, dist)
+        @test isapprox(location(dw), 5.0, atol=0.02)
+        @test isapprox(scale(dw), 3.0, atol=0.03)
```

Comment:
Can you test the weighted method gives the same answers as the unweighted method when `w` is a set of unit weights?